packages/gasket-plugin-webpack/docs/webpack.md

```diff
@@ -20,10 +20,10 @@ brittle, so handle with care. Also, the `webpack` instance is passed via context
 with the lifecycle, providing access to the [compatible Webpack plugins]
 of the version installed.
 
-### Via lifecycle files
+### Via custom app-level plugin
 
 ```js
-// lifecycles/webpack-config.js
+// my-custom-plugin/lifecycles/webpack-config.js
```

Comment:
Actually, this syntax is invalid now. I think the `Via plugin hooks` example covers the required pattern if you want to drop this `Via custom app-level plugin` block